server/constants/vars.js

```diff
@@ -20,6 +20,19 @@ export const DEFAULT_CHART_FILTERS = {
   ],
 }
 
+export const DEFAULT_CHART_FILTERS = {
+  chrcrit_part: [
+    { name: 'HC', value: TRUE_STRING },
+    { name: 'CHR', value: TRUE_STRING },
+    { name: 'Missing', value: TRUE_STRING },
+  ],
+  included_excluded: [
+    { name: 'Included', value: TRUE_STRING },
+    { name: 'Excluded', value: FALSE_STRING },
+    { name: 'Missing', value: FALSE_STRING },
+  ],
+}
```

Comment:
So subjects for which `chrcrit_part==1 && included_excluded==1` are counted only now?